src/scripts.js

```diff
@@ -27,15 +27,39 @@ const userSteps = document.querySelector('#userSteps');
 const userWater = document.querySelector('#userWater');
 const userSleep = document.querySelector('#userSleep');
 const userActivity = document.querySelector('#userActivity');
+const userEmail = document.querySelector('#userEmail');
+const userAddress = document.querySelector('#userAddress');
+const userStride = document.querySelector('#userStride');
+const userStepGoal = document.querySelector('#userStepGoal');
 
 //eventListeners go here
-window.addEventListener('load', displayName);
+window.addEventListener('load', displayUserInfoCard);
 //helper functions go here
 
-function displayName() {
+function displayUserInfoCard() {
```

Comment:
We need a refactor branch to update this function and all the ones below it to es6